ResearchKit/Common/ORKOrderedTask.m

```diff
@@ -1090,24 +1091,22 @@ + (ORKOrderedTask *)timedWalkTaskWithIdentifier:(NSString *)identifier
             
             ORKStepArrayAddStep(steps, step);
         }
-        
+
+        {
+            ORKTimedWalkStep *step = [[ORKTimedWalkStep alloc] initWithIdentifier:ORKTimedWalkTurnAroundStepIdentifier];
+            step.title = ORKLocalizedString(@"TIMED_WALK_INSTRUCTION_TURN", nil);
+            step.text = ORKLocalizedString(@"TIMED_WALK_INSTRUCTION_TEXT", nil);
+            step.spokenInstruction = step.title;
+            step.recorderConfigurations = recorderConfigurations;
+            step.distanceInMeters = 1;
+            step.shouldTintImages = YES;
+            step.image = [UIImage imageNamed:@"turnaround" inBundle:[NSBundle bundleForClass:[self class]] compatibleWithTraitCollection:nil];
+            step.stepDuration = timeLimit == 0 ? CGFLOAT_MAX : timeLimit;
```

Comment:
@umerkhan-apple This is a bit awkward, but I'm no longer working for Shazino, therefore I no longer have push access to the fork for this PR. Do you want me to essentially reapply these changes to a new fork and send a new PR, or do you prefer to merge this in and let me submit further improvements (including the one being discussed here) from the merged code?